src/composables/walletConnect.ts

```diff
@@ -83,16 +83,16 @@ export function useWalletConnect({ offscreen } = { offscreen: false }) {
       const gas = Number(params.gas);
       const senderId = toChecksumAddress(params.from);
       const recipientId = toChecksumAddress(params.to);
-      const isCoinTransfer = !!params.value; // `value` is present only when sending ETH
-      const tag = (params.data) ? Tag.ContractCallTx : Tag.SpendTx;
+      const isCoinTransfer = !params.data;
+      const tag = isCoinTransfer ? Tag.SpendTx : Tag.ContractCallTx;
       const modalProps: IModalProps = {
         protocol: PROTOCOLS.ethereum,
         app: { url, host: url ? new URL(url).hostname : '', name },
         tx: {
           amount: params.value ? +fromWei(params.value, 'ether') : 0,
           fee: gas * +(maxFeePerGas.value || 0),
           gas,
-          contractId: (isCoinTransfer) ? ETH_CONTRACT_ID : recipientId,
+          contractId: isCoinTransfer ? ETH_CONTRACT_ID : recipientId,
```

Comment:
I know that both options: with/without round brackets are acceptable. But I suggest to leave the brackets. It is a good habit as it makes the code more readable when eyeballing the code in hurry.

When you see this:
```
const a = something ? foo : bar;
```
You might mistake the ternary with regular assignment:
```
const a = something;
```

If you add brackets you immediately know that the line starts with ternary:
```
const a = (something) ? foo : bar;
```

This is a thing that really good dev taught me years ago and practice showed me that he was right :)

Reply:
I agree that this looks better, I just don't think I've seen it in our app. But maybe it's time we start using it